### PR TITLE
Test: update test file with separate functions for each unit test 

### LINKED ISSUE #16: Code for Box plot

### LINKED ISSUE #21: Test script for Boxplot

### PR DESCRIPTION
**Pull request by miyer26**

### CODE REVIEW

**Comment by miyer26** [comment]
General question, is it better to have a separate function for each test or for each type of test? My approach was the latter, but please advice what is the best practice. Thanks!

**Comment by ruben1dlg** [comment]
I had one function per test! But I guess both options are valid 😀

**Comment by miyer26** [comment]
As discussed, I have included tests to check for the raised errors in the main function. Thanks for bringing it up.

File: src/magmaviz/boxplot.py [hunk 1]
```diff
@@ -28,6 +31,10 @@ def boxplot(df, x, y, facet=False):
     """
 
     # checking that type of column name for x is a string
```
Comment (shyan0903) [inline src/magmaviz/boxplot.py]: This comment is not for the function defined one line below. Should be moved to the right place for clarity.

Comment (miyer26) [reply src/magmaviz/boxplot.py]: agreed, changes reflected in latest commit

**Comment by ruben1dlg** [comment]
Looks very good! I will approve it on my side! Let's wait for the rest of the team to approve it so we can merge it!

**Review by iamMoid — APPROVED** [review]
Great work. Looks good to me.

**Review by ruben1dlg — APPROVED** [review]

**Review by shyan0903 — APPROVED** [review]

**Pull request merged**